src/pages/candidate-details/candidate-details.ts

```diff
@@ -36,6 +41,7 @@ export class CandidateDetailsPage extends BasePageComponent implements OnInit, O
   pageState: CandidateDetailsPageState;
   subscription: Subscription;
   slotId: number;
+  slotChanged: boolean = false;
```

Comment:
Might be worth checking with the NgRX gurus whether we can put nav params (i.e. non-observables) into page state instead of as fields in the class.

Reply:
slotChanged was just following the way slotId is implemented. Personally I don't think the navParams should be in the pageState as they aren't coming from the store.  Will remove both and see if it has knock on effects

Reply:
Have moved both out of pagestate. No sideeffects